lib/ret/login.ex

```diff
@@ -22,4 +24,33 @@ defmodule Ret.Login do
     |> unique_constraint(:identifier_hash)
     |> put_assoc(:account, account)
   end
+
+  def update_identifier_hash(%{old_email: old_email, new_email: new_email} = _) do
+    old_identifier_hash = Account.identifier_hash_for_email(old_email)
+    new_identifier_hash = Account.identifier_hash_for_email(new_email)
```

Comment:
Consider inlining both of these variables. They aren’t reused and the names don’t aid understanding.

Reply:
Resolved in https://github.com/mozilla/reticulum/pull/636/commits/733d0dabda589d475c944c66b7c5a03c34529820